lib/cmds/space_cmds/environment_cmds/list.js

```diff
@@ -0,0 +1,58 @@
+import Table from 'cli-table2'
+
+import { getContext } from '../../../context'
+import { assertLoggedIn, assertSpaceIdProvided } from '../../../utils/assertions'
+import { handleAsyncError as handle } from '../../../utils/async'
+import { createManagementClient } from '../../../utils/contentful-clients'
+import { log } from '../../../utils/log'
+import paginate from '../../../utils/pagination'
+
+export const command = 'list'
+
+export const desc = 'List your space environments'
+
+export const builder = (yargs) => {
+  return yargs
+    .usage('Usage: contentful space environment list')
+    .option('space-id', {
+      describe: 'ID of the space the environment will belong to',
+      type: 'string'
+    })
+    .epilog([
+      'See more at:',
+      'https://github.com/contentful/contentful-cli/tree/master/docs/space/environment/list',
+      'Copyright 2018 Contentful, this is a BETA release'
+    ].join('\n'))
+}
+
+export const aliases = ['ls']
+
+async function environmentList (argv) {
+  await assertLoggedIn()
+  await assertSpaceIdProvided(argv)
```

Comment:
Maybe at some point, we need to use [Middleware](https://github.com/yargs/yargs/blob/master/docs/advanced.md#middleware)

Reply:
yeah till then lets keep the old, not that ideal but working, pattern